backend/src/main/java/com/huellapositiva/application/controller/ProposalApiController.java

```diff
@@ -96,7 +97,7 @@ public void createProposal(@RequestPart("dto") MultipartFile dtoMultipart,
                                @AuthenticationPrincipal String contactPersonEmail,
                                HttpServletResponse res) throws IOException {
         ProposalRequestDto dto = objectMapper.readValue(dtoMultipart.getBytes(), ProposalRequestDto.class);
-        dto.setStatus(PUBLISHED.getId());
+        dto.setStatus(REVIEW_PENDING.getId()); // Setting the status when a proposal is created the first time.
```

Comment:
Setting the default proposal status seems to be responsibility of the register proposal action rather than the controller

Reply:
OK, yes, you are right. We have just changed it. Thanks!